src/main/java/com/bravo/user/model/filter/UserFilter.java

```diff
@@ -7,9 +7,40 @@
 @Data
 public class UserFilter {
 
+  public Set<String> getIds() {
+    return ids;
+  }
+
   private Set<String> ids;
   private Set<String> firstNames;
   private Set<String> lastNames;
   private Set<String> middleNames;
+  private Set<Integer> phoneNumbers;
+
+  public Set<String> getFirstNames() {
+    return firstNames;
+  }
+
+  public Set<String> getLastNames() {
+    return lastNames;
+  }
+
+  public Set<String> getMiddleNames() {
+    return middleNames;
+  }
+
+  public Set<Integer> getPhoneNumbers() {
+    return phoneNumbers;
+  }
+
+  public Integer getPhoneNumberLength() {
+    return phoneNumberLength;
+  }
+
+  public DateFilter<LocalDateTime> getDateFilter() {
+    return dateFilter;
+  }
+
+  private final Integer phoneNumberLength = 10;
```

Comment:
1. This value doesn't belong here. It should be declared in a constants or validator file. 
2. This variable should be `static` so it can be referenced without an instance of the object.

Reply:
I agree and I made a comment to your point. I just did not have time to either add a validator or be familiar enough w/your code to know the proper place to put this. static of course should have been added.